### PR TITLE
Update create-swap-file-linux-vm.md

### PR DESCRIPTION
**Pull request by markusries**
Committing changes for updating swap file document.   Had a peer review.

# Pull request guidance

Thank you for submitting your contribution to our support content! Our team works closely with subject matter experts in CSS and PMs in the product group to review all content requests to ensure technical accuracy and the best customer experience. This process can sometimes take one or more days, so we greatly appreciate your patience.

We also need your help in order to process your request as soon as possible:

- We won't act on your pull request (PR) until you type "**#sign-off**" in a new comment in your pull request (PR) to indicate that your changes are complete.

- After you sign off in your PR, the article will be tech reviewed by the PM or SME if it has more than minor changes. Once the article is approved, it will undergo a final editing pass before being merged.

### CODE REVIEW

**Comment by AmandaAZ** [comment]
@markusries @mabicca I made some editorial changes for better readability, such as restructuring the "Create a SWAP partition" part and deleting redundant content. Could you review this PR again? Let me know if there is any problem.

File: support/azure/virtual-machines/create-swap-file-linux-vm.md [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
+To create a SWAP partition on Azure Linux VMs, you need to set up cloud-init to automatically create it on the ephemeral (resource) disk of the VM. The resource disk is mounted under `/mnt` by default. It's located on the physical server where the Linux VM is hosted and has lower latency. It isn't recommended to create SWAP partitions on OS disks or data disks that might impact the performance of the operating system and apps. It's important to remember that the resource disk should never be used to store regular data since it's only temporary storage. When a VM is moved to another host or stopped/deallocated, any data written to this disk will be wiped. It's recommended to use the resource disk only for data that can be removed such as SWAP and caching files. For more information, see [Temporary disk](/azure/virtual-machines/managed-disks-overview#temporary-disk).
```
Comment (mabicca) [inline support/azure/virtual-machines/create-swap-file-linux-vm.md:15]: <!NOTE: These steps will be adding a SWAP file under the SWAP partition.

**Review by mabicca — COMMENTED** [review]
Added note about the steps:

NOTE: These steps will be adding a SWAP file under the SWAP partition.

**Comment by AmandaAZ** [comment]
> Added note about the steps:
> 
> NOTE: These steps will be adding a SWAP file under the SWAP partition.

@mabicca I added such a note under the "[Create a SWAP partition](https://github.com/MicrosoftDocs/SupportArticles-docs/blob/1621a0419696df9f40c37856addbb13a6bbb3808/support/azure/virtual-machines/create-swap-file-linux-vm.md#create-a-swap-partition)" section. Let me know if any problem. You can preview the note/article from [SupportArticles-docs/support/azure/virtual-machines/create-swap-file-linux-vm.md at 1621a0419696df9f40c37856addbb13a6bbb3808 · MicrosoftDocs/SupportArticles-docs (github.com)](https://github.com/MicrosoftDocs/SupportArticles-docs/blob/1621a0419696df9f40c37856addbb13a6bbb3808/support/azure/virtual-machines/create-swap-file-linux-vm.md).

File: support/azure/virtual-machines/create-swap-file-linux-vm.md (hunk 1 above)
Comment (AmandaAZ) [reply support/azure/virtual-machines/create-swap-file-linux-vm.md:16]: The note is added.

**Review by mabicca — APPROVED** [review]

**Review by mabicca — APPROVED** [review]

**Review by mabicca — APPROVED** [review]

**Review by mabicca — APPROVED** [review]

**Pull request merged**